packages/api/package.json

```diff
@@ -77,23 +77,23 @@
     "node": ">=18.0.0"
   },
   "dependencies": {
-    "@web5/agent": "0.3.8",
+    "@web5/agent": "workspace:*",
```

Comment:
intentional? probably needs a revert

Reply:
@shamilovtim, can you elaborate why this should be reverted? I did this intentionally with the hope that we don't need to keep having two phase version update of doing `agent` followed by the `api`.

Reply:
confused: this is an active package that gets published to npm. isn't this an invalid package.json.
isn't it the case that you can't ship this change to end users? also, what about changesets?

Reply:
pnpm will take care of translating the `workspace:*` into the right version.

Reply:
i'm not following how this is any easier than predictable versioning that we can reason about with a historical publish record that matches version control

Reply:
The intent is that if a dependent package introduces a breaking change to downstream mono repo, CICD will not let the PR through. Nor would one need to remember to hunt and update downstream packages to depend on the new version.